code/lib/cli/src/generate.ts

```diff
@@ -216,6 +216,10 @@ command('dev')
   )
   .option('--force-build-preview', 'Build the preview iframe even if you are using --preview-url')
   .option('--docs', 'Build a documentation-only site using addon-docs')
+  .option(
+    '--initial-path [path]',
```

Comment:
This has to be added to the angular schema as well, if we aim to support this CLI flag there as well. (https://github.com/storybookjs/storybook/blob/next/code/frameworks/angular/src/builders/start-storybook/schema.json, https://github.com/storybookjs/storybook/blob/next/code/frameworks/angular/src/builders/start-storybook/index.ts)

Reply:
Thanks!